src/appleseed/foundation/image/genericprogressiveimagefilereader.cpp

```diff
@@ -38,17 +38,12 @@
 #include "foundation/image/tile.h"
 
 // OpenImageIO headers.
+#include "foundation/platform/_beginexrheaders.h"
 #include "foundation/platform/_beginoiioheaders.h"
```

Comment:
If OpenImageIO includes OpenEXR header files, should `_beginoiioheaders.h` include `_beginexrheaders.h`?

Reply:
Yes, that make sense. I will try to add `#include "foundation/platform/_beginexrheaders.h"` inside `_beginoiioheaders .h`.